### DISCUSSION TITLE
Remove Bluecord from the page

### DISCUSSION THREAD

**Discussion by DrivingInMyCarRightAfterABeer** [Discord-Client-Encyclopedia-Management/Discord3rdparties#94] [Ideas]
If Bluecord creator is known for scamming and accounts stealing, why don't you just remove his mod entirely from this page? I mean he has stolen many Kik accounts before and turned them into his botnets. It would be better if we just remove Bluecord entirely on this page

**Comment by lexyclear** [comment]
for historical reasons and as a warning we keep it here so people avoid it.
